javascript/reconcile.js

```diff
@@ -42,6 +44,11 @@ export default (rootElement, newState, keyAttribute) => {
       return false
     }
 
+    // Leave elements that are marked with the `permanentAttribute` alone.
+    if (fromElement.hasAttribute(permanentAttribute)) {
+      return false
```

Comment:
If this is what permanent attribute does (not update the DOM after initialization), I am wondering what the use case for this feature will be?

Unlike in the turbolinks example, if you wanted to have a shopping cart update on the page but not update the rest of the DOM (or vice versa), you would use a motion component to listen for updates to the cart and you would update it.

We are also already checking to see if the key is the same before updating, so there is a feature already where a user could choose not to update an element (if that was desirable).

So the only reason that I can think of that you would have an element that should not update is if the client side stored state, and that would have to be in JS, and I don't think Motion quite supports the uses cases i can think of.

So I approve this PR, but I am not sure what the use case is.  (Maybe I am not thinking of some that do exist.)

I'm interested in others thoughts.

Reply:
This is a fair point.

I was imaging the use-case for this would be using other JavaScript libraries in Motion, but upon further thought, until we give some way to call JavaScript from Motion (maybe some kind of effects system), it's not clear that this is very useful.